<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,4,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Statistics Summary</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4,1,2,1,1,1,3,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="statusLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Number of users participating:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="usersLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Number of real items:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="itemsLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Number of viable real items:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="viableLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Number of trades:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tradesLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Iterations completed:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="iterLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,3,1,1,3,1,1">
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="urlButton">
          <property name="text">
           <string>Open URL</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="disp">
      <property name="font">
       <font>
        <family>Lucida Console</family>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
